example/config.yml

```diff
@@ -56,3 +56,89 @@ collections: # A list of collections the CMS should be able to edit
             fields:
               - {label: "Name", name: "name", widget: "string"}
               - {label: "Description", name: "description", widget: "markdown"}
+
+  - name: "kitchenSink" # all the things in one entry, for documentation and quick testing
+    label: "Kitchen Sink"
+    folder: "_sink"
+    create: true
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Boolean", name: "boolean", widget: "boolean"}
```

Comment:
Might want to set the "default" setting on the boolean controls to test that as well.